Source/DafnyRuntime/DafnyRuntime.py

```diff
@@ -1,6 +1,6 @@
 class _dafny:
     def print(toPrintVal):
         if type(toPrintVal)== bool:
-            print("true" if toPrintVal else "false")
+            print("true" if toPrintVal else "false",end="")
```

Comment:
"end" to keep all printed statements in the same line as newLine operators are printed by compiler.